specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/examples/DataAssets_Get.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-567890abcdef",
+    "resourceGroupName": "SampleResourceGroup",
+    "workspaceName": "Workspace1",
+    "dataAssetName": "DataAsset1",
+    "api-version": "2020-05-04-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 13 Sep 2017 18:04:32 GMT",
+        "x-ms-request-id": "d5496da4-9c52-402f-b067-83cc9ddea888",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-ratelimit-remaining-tenant-reads": "14999",
+        "x-ms-correlation-request-id": "25c78f97-0b0a-4fe9-ad39-883a482265cd"
+      },
+      "body": {
+        "properties": {
+          "dataAssetId": "a08f184b-0567-890a-ba22-a1199336d226",
+          "description": "Data of DataSet1"
+        },
+        "id": "/subscriptions/12345678-1234-1234-1234-567890abcdef/resourceGroups/SampleResourceGroup/providers/Microsoft.DataCollaboration/workspaces/Workspace1/dataAssets/DataAsset1",
+        "name": "DataAsset1",
+        "type": "Microsoft.DataCollaboration/DataAsset"
```

Comment:
The type is not accurate in the examples for any nested resource.  This one should be "Microsoft.DataCollaboration/workspaces/dataassets" for example